packages/tests/tests/filter/multiple_pubsub.node.spec.ts

```diff
@@ -178,3 +180,177 @@ describe("Waku Filter V2: Multiple PubsubTopics", function () {
     }
   });
 });
+
+describe("Waku Filter V2 (Autosharding): Multiple PubsubTopics", function () {
+  // Set the timeout for all tests in this suite. Can be overwritten at test level
+  this.timeout(30000);
+  let waku: LightNode;
+  let nwaku: NimGoNode;
+  let nwaku2: NimGoNode;
+  let subscription: IFilterSubscription;
+  let messageCollector: MessageCollector;
+
+  const customContentTopic1 = "/waku/2/content/test.js";
```

Comment:
```suggestion
  const customContentTopic1 = "/waku/2/content/null";
```

Just to keep in line with expected content topic format. or you could use `utf8` too.

Reply:
updated all to use `utf8` instead of `test.js`